km61/Kulish_Oleh/laboratory1.sql

```diff
@@ -1,2 +1,128 @@
 -- LABORATORY WORK 1
 -- BY Kulish_Oleh
+
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+робити вибірки з таблиць
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER OLEH_KULISH(
+  DEFAULT TABLESPACE "USERS",
+  TEMP TABLESPACE "TEMP"
+); 
+
+ALTER USER OLEH_KULISH QUONT 100M ON USERS;
+
+GRANT "CONECT" TO OLEH_KULISH;
+
+GRANT ALTER ANY /*Якесь слово*/ TO OLEH_KULISH;
+
+
+
+
+
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Громадянин України має власне житло та автомобіль.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE TABLE PEOPLES (people VARCHAR(20));
+ALTER TABLE PEOPLES 
+    ADD CONSTRAINT people_pk PRIMARY KEY (people);
+
+CREATE TABLE HOMES (home VARCHAR(25));
+ALTER TABLE HOMES
+    ADD CONSTRAINT home_pk PRIMARY KEY (home);
+
+CREATE TABLE CARS (car VARCHAR(25));
+ALTER TABLE CARS
+    ADD CONSTRAINT car_pk PRIMARY KEY (car);
+
```

Comment:
The logic of Oleh's answer is wrong, he needed no create tablets "car" and "house" then make alters of them, and after this create tablet "person", which contains previous tablets. Then with alter table and constraint construction he was able to create foreign keys.